src/styles/components/_carousel.scss

```diff
@@ -1,5 +1,39 @@
 $mc-carousel-centered-padding: 100px !default;
 
+.slick-dots {
+  /* stylelint-disable */
+  display: flex !important;
+  /* stylelint-enable */
+  justify-content: center;
+  padding: 25px 0;
+
+  li {
+    line-height: 0;
+  }
+
+  button {
+    background: $mc-color-gray-400;
+    width: 6px;
+    height: 6px;
+    border-radius: 6px;
+    margin: 0 4px;
+    overflow: hidden;
+    text-indent: 9999em;
+    cursor: pointer;
+    transition: background 250ms ease;
+    outline: 0;
+
+    &:hover {
+      background: $mc-color-gray-300;
+    }
+  }
+
+  .slick-active button {
```

Comment:
I'm not sure if we can override slick's default class for active or not. [Documentation is here](http://kenwheeler.github.io/slick/) for slick slider.

Reply:
Same comment here re: wrapping class in namespace.